src/scss/pages/_homePage.scss

```diff
@@ -0,0 +1,125 @@
+.Section {
+  &-update {
+    padding: 120px 0;
+    height: 560px;
+    background-image: url(/public/avengersinfinitywar.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  &-gallery {
+    padding: 120px 0 90px;
+  }
+
+  &-gaming {
+    padding: 120px 0;
+    background-image: url(/public/playstation-cover.png);
+    background-size: cover;
+  }
+
+  &-latest-blog {
+    padding: 80px 0 120px;
+  }
+}
+
+.Gallery {
```

Comment:
Per each root class you should create a file.

Reply:
👍🏼